main.rb

```diff
@@ -0,0 +1,50 @@
+require_relative './app'
+
+class Main
+  def self.home_page
+    puts 'Please enter a number to select an option:'
+
+    @content = {
+      '1' => 'List all books',
+      '2' => 'List all people',
+      '3' => 'Create a person (student or teacher)',
+      '4' => 'Create a book',
+      '5' => 'Create a rental',
+      '6' => 'List all rentals for a given person id',
```

Comment:
- [ ] Your app throws an error in an attempt to list all rentals for a given person when an ID is selected. Kindly ensure to improve upon this aspect:

**List rental error:**
![image](https://user-images.githubusercontent.com/12745474/170666666-57b04975-245e-4739-81ef-694d8aa06d32.png)

Reply:
Thanks for the review issue is resolved